scripts/base/init-bare.zeek

```diff
@@ -3048,6 +3048,20 @@ type dns_svcb_rr: record {
 	target_name: string;	##< Target name, the hostname of the service endpoint.
 };
 
+## A NAPTR record.
+##
+## See also RFC 2915 - The Naming Authority Pointer (NAPTR) DNS Resource Record.
+##
+## .. zeek:see:: dns_NAPTR_reply
+type dns_naptr_rr: record {
+	order: count;        ##< Order in which to process NAPTR records.
+	preference: count;   ##< Preference specifying processing order for *equal* :zeek:field:`dns_naptr_rr$order` fields.
+	flags: string;       ##< Flags to control rewriting. E.g. "u", "a", "s" or "p".
```

Comment:
Are flags case-insensitive? From [the RFC](https://www.rfc-editor.org/rfc/rfc2915):

> A <character-string> containing flags to control aspects of the rewriting and interpretation of the fields in the record.  Flags are single characters from the set [A-Z0-9].

Then later they mention upper case flags and other places seem to use upper case:

>  At this time only four flags, "S", "A", "U", and "P", are defined.

But the test below has a lower case flag, so it's confusing to me. Do you know if it's just not case sensitive?

Reply:
Oh wow Evan, nice sleuthing, I didn't read or question too closely.

From the follow-up RFC 3403:

       A <character-string> containing flags to control aspects of the
      rewriting and interpretation of the fields in the record.  Flags
      are single characters from the set A-Z and 0-9.  ***The case of the
      alphabetic characters is not significant. *** The field can be empty.

https://datatracker.ietf.org/doc/rfc3403/

I think we should keep the casing just to not loose that information.